guide/english/book-recommendations/python/index.md

```diff
@@ -18,7 +18,6 @@ title: Books on Python Programming Language
 Paid (Free ebook available) book which covers the basics of python. It's designed to get you started with python language and become familiar with its syntax and workings by the time you complete the book. 
 
 - [Website](https://learnpythonthehardway.org/) (Buy from the creator | Includes video lessons)
-- [E-book](https://learnpythonthehardway.org/python3/) (free)
```

Comment:
It is free, it's the e-book part that is wrong, since it's all on the website and not in an epub or something.

Reply:
I opened the website, clicked next chapter for a few times, and encountered text saying I needed to buy it. The link is just a 'sample'. 

Could you try doing that yourself?

Reply:
True, but it's still a valuable resource, so it'd be better to just remove the "free" tag.

Reply:
It's a different style, like the other learn X the hard way resources, forcing you to do a lot yourself, but that's it.